### PR TITLE
Corrected regression in the right-click Convert to numeric option

### LINKED ISSUE #9020: Correct regression in the right-click Convert to numeric option
@N-thony I hope this will be a simple correction - and maybe you need to do it, so it can be done before the next release.

@Patowhiz first brought it to my attention, and I am not sure when it happened.

In the survey data make (say) the fert variable into a character, and then change (say) zero for the letter o somewhere.

Now try the right-click and convert to numeric.  It used to give a popup and say one value was non-numeric and what do you wish to do?  Now it simply adds labels to the variable.  I am not sure what happens to the o.  But I'd like to get the popup back!

I don't think this was @derekagorhom change with the time series data, because that was too recent.  It happended a few weeks ago.

### PR DESCRIPTION
**Pull request by N-thony**
Fixes #9020 
@rdstern have a look

### CODE REVIEW

**Review by rdstern — APPROVED** [review]
@N-thony great - well done.
@Patowhiz I am relieved!  Can you test, check and merge if ok?

File: instat/ucrDataView.vb [hunk 1]
```diff
@@ -661,23 +661,18 @@ Public Class ucrDataView
                 GetCurrentDataFrameFocus().clsPrepareFunctions.ConvertToNumeric(strColumn, True)
             Else
                 Dim bCheckLabels As Boolean = GetCurrentDataFrameFocus().clsPrepareFunctions.CheckHasLabels(strColumn)
-                If bCheckLabels Then
```
Comment (Patowhiz) [inline instat/ucrDataView.vb:664]: @N-thony are you able to know why this check was added? I don't remember merging this functionality. I see removing the if check has fixed the bug, but would be good to check why it exists.

File: instat/ucrDataView.vb [hunk 2]
```diff
@@ -661,23 +661,18 @@ Public Class ucrDataView
...
-                            Continue For
-                    End Select
-                Else
+                frmConvertToNumeric.SetDataFrameName(GetCurrentDataFrameFocus().strName)
+                frmConvertToNumeric.SetColumnName(strColumn)
+                frmConvertToNumeric.CheckLabels(bCheckLabels)
+                frmConvertToNumeric.SetNonNumeric(iNonNumericValues)
+                frmConvertToNumeric.ShowDialog()
+                ' Yes for "normal" convert and No for "ordinal" convert
+                If frmConvertToNumeric.DialogResult = DialogResult.Yes Then
```
Comment (Patowhiz) [inline instat/ucrDataView.vb:670]: @N-thony I think the Select  Case is clearer, so I recommend retaining it.

**Review by Patowhiz — COMMENTED** [review]
@N-thony thanks for quickly addressing this. Just a few questions and suggestions then we can merge.

**Review by Patowhiz — APPROVED** [review]
Thanks @N-thony for your response and looking into this in detail.

@rdstern this PR reverts PR #8920 (which I merged) and re-opens issue #8917.

@rdstern this is a good example that shows our test weaknesses. Moving forward, **for features that affect existing features**, I suggest having more than 1 person that would test as end users independently. I can be that second person though it may end up taking more time when it comes to merging PRs like these.

@rdstern once you approve this, I'll merge.

**Review by rdstern — APPROVED** [review]

**Pull request merged**